parserscripts/pfam_db.py

```diff
@@ -93,26 +102,29 @@ def getValues(url):
 
     return result
 
+
 def main():
-    translations = extractTranslations(filePath)
-    cds = extractCDS(filePath)
+    translations = extract_translations(file_path)
+    cds = extract_cds(file_path)
 
     for i, translation in enumerate(translations):
-        print("Getting translation " + str(i + 1) + "/" + str(len(translations)) + "...")
-        url = getTranslationURL(translation)
-        result = getValues(url)
+        print("Getting translation " + str(i + 1) + "/" + str(
+            len(translations)) + "...")
```

Comment:
You've got parentheses already; break before `str`, and align with opening parenthesis.

Reply:
Damn, another one. I think these two (`str` breaks) were caused by PyCharm's `Reformat File`. I agree with your statement and I will fix it ASAP.